src/service/forumService.js

```
@@ -40,6 +40,15 @@ class ForumService {
       throw error;
     }
   }
+
+  static async getAllQuestionsWithDefaultPagination() {
+    try {
+      const response = await axiosInstance.get(`/question/pagination`);
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)